langchain/chains/llm_summarization_checker/base.py

```diff
@@ -0,0 +1,130 @@
+"""Chain for summarization with self-verification."""
+
+from typing import Dict, List
+
+from pydantic import BaseModel, Extra
+
+from langchain.chains.base import Chain
+from langchain.chains.llm import LLMChain
+from langchain.chains.sequential import SequentialChain
+from langchain.llms.base import BaseLLM
+from langchain.prompts.prompt import PromptTemplate
+
+CREATE_ASSERTIONS_PROMPT = PromptTemplate.from_file(
```

Comment:
the text files sadly arent going to be automatically included in the package binary - will need to add that to setup.py (or change to just being .py files)

Reply:
I've found managing prompts in separate unstructured text to be extremely useful for iterating and testing. 

At least in pycharm, copilot works much better for prompting in a txt or md fine than in a `prompt.py`, for example. 
 Not sure why, but probably because it's not getting confused with the context of all the other syntax.

Also, in reviews with non-technical people, I've found eyes glaze over less and more people are able to contribute without mucking around with python/yaml/json.  So i've defaulted to using prompt dirs.  

That being said, if it's a ton of extra work, I can refactor.

Reply:
nah ill work on adding. shouldnt be that bad as long as all in same file